api/routes.go

```diff
@@ -144,8 +144,8 @@ func (api *API) routes(auth *Authentication, tokenHandler *TokenHandler) {
 	router.PATCH("/data-model/fields/:fieldID", api.UpdateDataModelField)
 	router.DELETE("/data-model", api.DeleteDataModel)
 	router.GET("/data-model/openapi", api.OpenAPI)
-	router.POST("data-model/pivots", api.createDataModelPivot)
-	router.GET("data-model/pivots", api.listDataModelPivots)
+	router.POST("/data-model/pivots", api.createDataModelPivot)
```

Comment:
oh, did that make a difference ? 🤔 
It was working on postman at least

Reply:
I think not, it was just uniformisation

Reply:
ah great, thanks :)